src/panels/lovelace/cards/picture-elements/create-configured-hui-element.ts

```diff
@@ -0,0 +1,25 @@
+import { HomeAssistant } from "../../../../types";
+import { LovelaceElement, LovelaceElementConfig } from "../../elements/types";
+import { createHuiElement } from "../../common/create-hui-element";
+
+export function createConfiguredHuiElement(
+  elementConfig: LovelaceElementConfig,
+  hass?: HomeAssistant
```

Comment:
Do not take in `hass`. None of our createXElement functions do so. It's up to caller to assign hass when available.